.github/workflows/htmcore.yml

```diff
@@ -92,6 +92,7 @@ jobs:
         path: build/Release/distr/dist
 
     - name: Test python package is working
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```

Comment:
fixing an unrelated CI failuer introduced in other PR, this breaks all current PRs